File: internal/content/testdata/bench/javascript/find-celebrity-in-party.js
function find_celebrity(knows_matrix) {
    const n = knows_matrix.length;
    let candidate = 0;
    for (let i = 1; i < n; i++) {
        if (knows_matrix[candidate][i]) {
            candidate = i;
        }
    }
    for (let i = 0; i < n; i++) {
        if (i !== candidate) {
            if (knows_matrix[candidate][i] || !knows_matrix[i][candidate]) {
                return -1;
            }
        }
    }
    return candidate;
}

File: internal/content/testdata/bench/java/find-celebrity-in-party.java
class Solution {
    public long find_celebrity(boolean[][] knows_matrix) {
        int n = knows_matrix.length;
        int candidate = 0;
        for (int i = 1; i < n; i++) {
            if (knows_matrix[candidate][i]) {
                candidate = i;
            }
        }
        for (int i = 0; i < n; i++) {
            if (i != candidate) {
                if (knows_matrix[candidate][i] || !knows_matrix[i][candidate]) {
                    return -1;
                }
            }
        }
        return candidate;
    }
}
